include/core/ib_particle.h

```diff
@@ -360,6 +368,8 @@ class IBParticle
   // Location of the pressure reference point relative to the center of the
   // particle. This point is used to define a constant on the pressure.
   Point<dim> pressure_location;
+
+  std::string particle_type;
```

Comment:
Can you add a comment to explain what this is?
Why is particle_type a string? why not an enum?